src/test/resources/json/timestamped/part2-rerun-failed.json

```diff
@@ -0,0 +1,235 @@
+[
```

Comment:
does this have to be such big?
it should contain only those features and scenario which are required for tests

Reply:
I just want to make sure that the ReportParser handles a real report correctly. The biggest json which is added here is 13Kb. This one is 11Kb. Not too much to check the parser.)

Reply:
I'd prefer to have small pieces so in case of regressions it is easier to find what is really important there and what should be fixed. Remove all parts which are not relevant to new feature

Reply:
Hi! done! I did the files small as possible, there is no extra data anymore.
Also, I updated the javadoc for the test class.